frontend/src/components/pages/DocumentDirectory.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const DocumentDirectory = () => {
```

Comment:
This line is causing a lint error because it does not have a return type. This should fix the issue: 
```
const DocumentDirectory = (): React.ReactElement => {
```